.github/workflows/pex-build-upload.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: Build Pex
         run: |-
           mkdir -p dist
-          pex .[gojsonnet] -r requirements.txt --python-shebang='#!/usr/bin/env python3' --python=python3.7 --python=python3.8 --python=python3.9 -m kapitan -o dist/kapitan.linux-x86_64.pex
+          pex .[gojsonnet] -r requirements.txt --python-shebang='#!/usr/bin/env python3' --python=python3.8 --python=python3.9 --python=python3.10 -m kapitan -o dist/kapitan.linux-x86_64.pex
```

Comment:
Are we dropping testing for python 3.7 then?

Reply:
It has been already dropped, but we forgot to change it here. I think the action isn't even used anymore, but just I wanted to keep it up to date for potential future reuse.